### DISCUSSION TITLE
Automatically update prisma model and client on change in database

### DISCUSSION THREAD

**Discussion by HAPAAK** [prisma/prisma#18889] [Q&A] [answered]
When we run prisma db pull command, prisma introspects the database schema and populates the model.prisma and running prisma generate client will allow us to query the database.
But, I have noticed that whenever I added new tables, updated column or perform some drop operation. The changes are not reflected, I have to manually run the prisma db pull and prisma generate command and restart the server, to reflect the changes.
How do I make thing automatic for any schema changes in database, does prisma provides something that helps on this process? I have connection to multiple databases like mongodb, postgres and sql.

**Comment by nurul3101** [comment, marked as answer]
Hello @HAPAAK 👋 

At the moment, you would need to invoke `npx prisma db pull` command anytime you manually add the tables to your database. 

Other way would be to create tables through [Prisma Migrate](https://www.prisma.io/docs/concepts/components/prisma-migrate) so that prisma can be aware of the changes that you make.

**Reply by HAPAAK** [reply]
Yeah, right now. Anytime changes happen in database, I am manually invoking those commands.
Is it something that is being prepared for the future release?  

Anyway, thank you for the answer.

**Reply by nurul3101** [reply]
It's not on the immediate [roadmap](https://prismaio.notion.site/Prisma-ORM-Roadmap-50766227b779464ab98899accb98295f) yet.

**Reply by Tokitaka** [reply]
Hello @nurul3101 😆

Was there any update for this issue? Currently suffering same stuff ..
